src/components/tx/SendTx/CreateTx/create.js

```diff
@@ -92,56 +92,6 @@ const CreateTx = connect(
     onCancel: () => {
       ownProps.goDashboard();
     },
-    onSubmit: (data) => {
```

Comment:
Tx creation should be located in CreateTx component

Reply:
this wasn't actually modifying anything, all the tx create stuff happens in MultiCreateTx